.pipelines/cni/load-test-templates/restart-node-template.yaml

```diff
@@ -18,18 +18,16 @@ steps:
         clusterName=${{ parameters.clusterName }}
         make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${clusterName}
         make -C ./hack/aks azcfg AZCLI=az REGION=$(LOCATION)
-        cd test/integration/load
-        # Capture a scaledown, scaling down to 50% of initial value
-        scale=$(( ${{ parameters.scaleup }} * ${{ parameters.nodeCount }} / 2))
-        echo "Scaling the pods down to $(( $scale / ${{ parameters.nodeCount }} )) per node"
-        REPLICAS=$scale OS_TYPE=${{ parameters.os }} SKIP_WAIT=true go test -count 1 -timeout 30m -tags load -run ^TestScaleDeployment$ -tags=load
-        cd ../../../
+
         echo "Restarting the nodes"
         for val in $(az vmss list -g MC_${clusterName}_${clusterName}_$(LOCATION) --query "[].name" -o tsv); do
           make -C ./hack/aks restart-vmss AZCLI=az CLUSTER=${clusterName} REGION=$(LOCATION) VMSS_NAME=$val
         done
-        cd test/integration/load
-        REPLICAS=$scale OS_TYPE=${{ parameters.os }} go test -count 1 -timeout 30m -tags load -run ^TestScaleDeployment$
+
+        if ! [ ${{ parameters.cni }} = 'cniv1' ]; then
+          echo "Allow CNS to reassign ip's to pods"
+          sleep 30s

```

Comment:
How did we come up with 30 seconds ?

Reply:
Was testing various times. I am going to change this to a rollout status with a timeout. K8s api is matching the restart, but we had no checks confirming the ds/deploy were ready and available.